applications/tari_merge_mining_proxy/src/proxy.rs

```diff
@@ -98,12 +98,17 @@ pub struct MergeMiningProxyService {
 }
 
 impl MergeMiningProxyService {
-    pub fn new(config: MergeMiningProxyConfig, block_templates: BlockTemplateRepository) -> Self {
+    pub fn new(
+        config: MergeMiningProxyConfig,
+        http_client: reqwest::Client,
```

Comment:
looks like this change needs to be propagated to the tests too

Reply:
Thanks @delta1